eureka-common/src/main/resources/application.properties

```diff
@@ -48,6 +48,28 @@ eureka.version.incremental=${parsedVersion.incrementalVersion}
 eureka.version.qualifier=${parsedVersion.qualifier}
 eureka.version.buildNumber=${parsedVersion.buildNumber}
 eureka.stage=${eureka.stage}
+
+eureka.common.callbackserver = https://localhost:8443
+eureka.common.demoMode =true
+eureka.common.ephiProhibited=true
+eureka.common.registrationEnabled=true
+eureka.common.aiwSiteUrl=http://aiw.sourceforge.net
+eureka.common.helpSiteUrl=http://aiw.sourceforge.net/help/v1-10
+eureka.common.organizationName=Emory University
+eureka.common.dataEndpoint=proxy-resource
+eureka.common.filterEndpoint=protected/jstree3_searchsystemlist
+eureka.support.uri=''
```

Comment:
Check to make sure empty string=null string where the support URI is used. A safer change is to comment out the support URI variables so they're documented but keep the same behavior as before.